SDKGenerator.njsproj

```diff
@@ -918,6 +928,9 @@
     <Folder Include="targets\LumberyardSdk\testingTemplate\Code\Tests\" />
     <Folder Include="targets\LumberyardSdk\testingTemplate\Code\" />
     <Folder Include="targets\LumberyardSdk\testingTemplate\Code\Source\" />
+    <Folder Include="targets\python\Archive\" />
+    <Folder Include="targets\python\source\" />
+    <Folder Include="targets\python\source\source\" />
     <Folder Include="targets\XPlatCppSdk\" />
```

Comment:
Not related to python SDK but <Folder Include="targets\XPlatCppSdk\" /> isn't in the right place for alphabetical ordering

Reply:
nvm this is auto-generated apparently 👍